cms/templates/videos_index.html

```diff
@@ -60,13 +60,15 @@ <h3 class="sr">${_("Page Actions")}</h3>
                 <h3 class="title-3">${_("Managing uploaded files")}</h3>
-                <p>${_("After a file uploads successfully, automated processing begins. After automated processing begins for a file it is listed under Previous Uploads as {em_start}In Progress{em_end}. When the status is {em_start}Complete{em_end}, edX assigns a unique video ID to the video file and you can add it to your course. If something goes wrong, the {em_start}Failed{em_end} status message appears. Check for problems in the file and upload a replacement.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("After a file uploads to edX successfully, automated processing, which takes {em_start}24 hours{em_end}, begins at edX and the file appears under Previous Uploads as {em_start}In Progress{em_end}. EdX assigns a unique video ID to each file as soon as possible so that you can add the videos to your course while other processing continues. After edX assigns a unique video ID to a file, its status on this page changes to  {em_start}Complete{em_end}. However, the external video host sites continue to process the files.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("{em_start}Important{em_end}: Processing by the external video host sites continues even after you see the {em_start}Complete{em_end} status on this page. It takes {em_start}24 hours{em_end} for your YouTube channel to list an uploaded video.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("The file statuses on this page refresh periodically. If something goes wrong, the file status changes to {em_start}Failed{em_end} status message appears. Check for problems in your original file and upload a replacement.").format(em_start='<strong>', em_end="</strong>")}</p>
                 <h3 class="title-3">${_("How do I get the videos into my course?")}</h3>
-                <p>${_("After processing is complete for the video file, you copy its unique video ID. On the Course Outline page, you create or locate a video component to play this video. Edit the video component to paste the ID into the Advanced {em_start}EdX Video ID{em_end} field.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("As soon as edX assigns a unique video ID to a file, you can add that video to a video component in your course. Copy the unique video ID. In another browser window, on the Course Outline page, create or locate a video component to play this video. Edit the video component and paste the ID into the Advanced {em_start}EdX Video ID{em_end} field. The video plays in the LMS as soon as one encoded file is available at one of the video host sites. You do not have to wait until all encoded files are available on both host sites.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
"The video plays as in the LMS as soon as..." -- in this sentence, I suggest "as soon as *an* encoded file is available at *any* of the ...sites". Saying "as soon as one encoded file is available" is slightly confusing -- sounds as if some number of files might be available at a single host site.